src/Avalonia.Controls/Shapes/Arc.cs

```diff
@@ -0,0 +1,148 @@
+using System;
+using Avalonia.Media;
+
+namespace Avalonia.Controls.Shapes
+{
+    public class Arc : Shape
+    {
+        ArcSegment _arcSegment;
+        PathFigure _arcFigure;
+        PathGeometry _arcGeometry;
+
+        PathGeometry _emptyGeometry = new PathGeometry()
+        {
+            Figures = 
+            {
+                new PathFigure()
+                {
+                    StartPoint = new Point(0, 0)
+                }
+            }
+        };
+
+        public static readonly StyledProperty<double> StartAngleProperty =
+            AvaloniaProperty.Register<Arc, double>(nameof(StartAngle), 0.0);
+
+        public static readonly StyledProperty<double> EndAngleProperty =
+            AvaloniaProperty.Register<Arc, double>(nameof(EndAngle), 0.0);
+
+        static Arc()
+        {
+            StrokeThicknessProperty.OverrideDefaultValue<Arc>(1);
+            AffectsGeometry<Arc>(BoundsProperty, StrokeThicknessProperty, StartAngleProperty, EndAngleProperty);
+        }
+
+        public Arc() : base()
+        {
+            _arcSegment = new ArcSegment()
+            {
+                SweepDirection = SweepDirection.Clockwise
+            };
+
+            _arcFigure = new PathFigure()
+            {
+                Segments = 
+                {
+                    _arcSegment
+                },
+                IsClosed = false
+            };
+
+            _arcGeometry = new PathGeometry()
+            {
+                Figures = 
+                {
+                    _arcFigure
+                }
+            };
+        }
+
+        public double StartAngle
```

Comment:
Definitely add comments for what this is. It's especially important to note that this is degrees instead of radians.

Reply:
Done...